Source/PluginProcessor.h

```diff
@@ -19,7 +19,7 @@
 //==============================================================================
 /**
 */
-class PinkTromboneAudioProcessor  : public AudioProcessor, public Timer
+class PinkTromboneAudioProcessor  : public AudioProcessor, public ADSR

```

Comment:
Why does PinkTromboneAudioProcessor inherit from ADSR? I'm almost 100% sure that it should not.